### PR TITLE
DDF-1397 Upgrades RRD4J

### PR DESCRIPTION
**Pull request by coyotesqrl**
#### What does this PR do?
Upgrades to latest RRD4J version.

#### Who is reviewing it? 

@brjeter @pklinef @Kjames5269 
#### Select relevant component teams: 
@codice/data 
#### Ask 2 committers to review/merge the PR and tag them here.

@clockard
@shaundmorris

#### How should this be tested?

- Full CI build
- Metrics should be verified

#### What are the relevant tickets?
[DDF-1397](https://codice.atlassian.net/browse/DDF-1397)

#### Checklist:
- [ ] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

#### Notes on Review Process
Please see [Notes on Review Process](https://codice.atlassian.net/wiki/spaces/DDF/pages/71946981/Pull+Request+Guidelines) for further guidance on requirements for merging and abbreviated reviews. 

#### Review Comment Legend:
- ✏️ (Pencil) This comment is a nitpick or style suggestion, no action required for approval. This comment should provide a suggestion either as an in line code snippet or a gist. 
- ❓ (Question Mark) This comment is to gain a clearer understanding of design or code choices, clarification is required but action may not be necessary for approval.
- ❗ (Exclamation Mark) This comment is critical and requires clarification or action before approval.

### CODE REVIEW

**Comment by coyotesqrl** [comment]
There are a couple of bundle definitions that include `Import-Package` instructions specific to rrd4j. Those may need to be updated for this latest version but I have not gotten to the point where I can test them.

- `catalog-core-metricsplugin`
- `catalog-core-sourcemetricsplugin`
- `platform-metrics-interceptor`
- `platform-metrics-reporting`
- `sample-metrics`

**Comment by coyotesqrl** [comment]
build now

**Comment by coyotesqrl** [comment]
build now

File: platform/metrics/platform-metrics-reporting/src/main/java/ddf/metrics/reporting/internal/rest/MetricsEndpoint.java [hunk 1]
```diff
@@ -74,11 +75,11 @@
   public static final String DEFAULT_METRICS_DIR =
       new AbsolutePathResolver("data" + File.separator + "metrics" + File.separator).getPath();
 
-  static final Map<String, Long> TIME_RANGES = new HashMap<String, Long>();
+  static final Map<String, Long> TIME_RANGES = new HashMap<>();
 
   private static final Logger LOGGER = LoggerFactory.getLogger(MetricsEndpoint.class);
 
-  private static final String METRICS_SERVICE_BASE_URL = "/internal/metrics";
+  private static final String METRICS_SERVICE_BASE_URL = "/services/internal/metrics";
```
Comment (coyotesqrl) [inline platform/metrics/platform-metrics-reporting/src/main/java/ddf/metrics/reporting/internal/rest/MetricsEndpoint.java]: This resolves an unrelated issue that was discovered during testing.

**Comment by blen-desta** [comment]
✅ Hero'd successfully - Metrics work

**Pull request merged**